WireAPI/Tests/WireAPITests/EventDecoding/EventDataResource.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+struct EventDataResource {
```

Comment:
Maybe It's easier to add something to identify this as test/mock?
```suggestion
struct EventDataMockResource {
```

Reply:
done 25d8903fbb19a28fffff4e8ed546277255cf03c9

Reply:
Unfortunately I can't find the change, maybe something went wrong?

Reply:
strange, clicking on the commit in the previous comment shows the change for me

Reply:
I made the change again 0c8af4a9fba6b8fa355d08181eb6b2ac036dfe34

Reply:
Hm yes, now it works. thanks!